assets/css/admin-notices.scss

```diff
@@ -64,19 +64,19 @@ $sensei-notice-icon-width: 30px;
 		position: static;
 	}
 
-	&.sensei-notice-error {
+	&--error {
 		border-left-color: $notice-error;
 	}
 
-	&.sensei-notice-warning {
+	&--warning {
 		border-left-color: $notice-warning;
 	}
 
-	&.sensei-notice-info {
+	&--info {
 		border-left-color: $notice-info;
 	}
 
-	&.sensei-notice-success {
+	&--success {
```

Comment:
I am a little bit concerned that we are reducing the css specificity here. Normally when I have made the `&.<same-prefix-again>` in the past it was because I needed to increase specificity for some reason.

Reply:
Is there a conflict that comes up from doing this? I thought we were doing this more generally, but happy to revert if we need specificity.

Reply:
I understand the @aaronfc's concern. I might be wrong, but checking the original PR, it seems it was just an implementation decision. https://github.com/Automattic/sensei/pull/4797

I checked it quickly with Twenty Twenty-One/Two/Three, Divi, and Course theme, in the home and in the course page. And it seems fine.